### PR TITLE
October 2022 Newsletter

### PR DESCRIPTION
**Pull request by davidfischer**
Screenshot 2022-11-04 at 14-20-54 EthicalAds Newsletter - October 2022

### CODE REVIEW

File: content/posts/newsletter-october-2022.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+These are the major features we released in October:
+
+* We released a [campaign calculator]({filename}../pages/advertisers-calculator.md)
+  to help advertisers budget for a campaign based on the responses they're hoping to achieve.
+* We did our first [privacy teardown of search engines]({filename}../posts/2022-privacy-teardown-search-engines.md).
+  Depending on the response we get, we think this could be a series of posts
+  to educate audiences on what to look for in services from a privacy perspective.
+* Our GPU trained contextual targeting model rolled out into production in October.
+  So far, the results are much better than the previous iteration.
+  There were some issues taking the model from development into production,
```
Comment (ericholscher) [inline content/posts/newsletter-october-2022.md:25]:
```suggestion
  There were some performance issues taking the model from development into production,
```

File: content/posts/newsletter-october-2022.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+As I expected, October was a very strong month for us and for our publishers.
+We usually see a seasonal slowdown around the holidays and between fewer ads shown over the Thanksgiving holiday
+in the US and other seasonal pullback, we expect November to be a quite bit weaker than October.
+
+We've definitely seen some advertisers tighten their budgets as well and that's further contributing to a weaker finish to the year.
+Usually advertising is back to full swing in January or February,
+but outside of the brief Covid crash in March 2020,
+we haven't operated through a true downturn.
+Regardless, we can weather an economic downturn if it comes.
```
Comment (ericholscher) [inline content/posts/newsletter-october-2022.md:50]: Any more context? This seems very concrete without any additional info. Not 100% sure what we'd say, but perhaps just that we we're still a small scrappy startup?

File: content/posts/newsletter-october-2022.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+
+
+## Upcoming features
+
+The major features in our upcoming roadmap for November:
+
+* While it's early November, we already announced our
```
Comment (ericholscher) [inline content/posts/newsletter-october-2022.md:58]: This should be at the top? Nobody cares when things are published, as long as they come out before the newsletter.

**Review by ericholscher — APPROVED** [review]

File: content/posts/newsletter-october-2022.md (hunk 2 above)
Comment (davidfischer) [reply content/posts/newsletter-october-2022.md:50]: This is just a little forward guidance on what to expect over the next few months. I know that the next few months are likely to be down months and I don't want that to be a surprise.

**Pull request merged**